doc/source/getting_started.rst

```diff
@@ -21,21 +21,6 @@ You can clone the current repository of SpiceEV to your local machine using:
 
 This tool just has an optional dependency on Matplotlib for plotting, an optional dependency on sphinx for the documentation and an optional dependency on pytest for testing. Everything else uses the Python (>= 3.6) standard library.
```

Comment:
I think now we use Python >= 3.8 (tests are for 3.8 and 3.10)

Reply:
fine as is since >= 3.8 is only used for tests.